docs/guide/design_engine.md

```diff
@@ -17,6 +17,11 @@ For assets, the system uses the Symfony Asset component with asset packages.
   Typically a directory containing templates. For example, templates located under `app/Resources/views/themes/my_theme`
   or `src/AppBundle/Resources/views/themes/my_theme` are part of `my_theme` theme.
 
+!!! caution
+
+    After creating a new folder in `Resources/views/themes/` you need to clear Symfony cache `bin/console cache:clear`,
```

Comment:
On second thought, this might be not clear to everyone, so maybe something like:
```
After creating a new folder either in `app/Resources/views/themes/` or your bundle `Resources/views/themes/` directory you need to clear Symfony cache (`bin/console cache:clear`),
```